### PR TITLE
added cypress test scripts

### PR DESCRIPTION
**Pull request by NileshPatel17**

### CODE REVIEW

**Comment by RoopeHakulinen** [comment]
This would be a great additions. I just spend quite some time figuring what should I write into the `scripts`. Thanks @NileshPatel17 !

**Comment by anthonymjones** [comment]
Hi @NileshPatel17! I'm definitely open to reintroducing the `package.json` scripts for those who want them. I'm happy to approve if you add a schematic option like what I did with `noBuilder` so that we're not adding them by default, but only for those who want them.

I am curious though if you are running it with the builder, is there a reason you're not running `ng e2e` or `ng run {project name}:cypress-run`?

**Comment by NileshPatel17** [comment]
added schematic option for cypress script. Let me know if you want any further changes. 
I am curious how can i set default value. As of now, if i press enter on prompt, it accept ```Yes``` by default.
```
"addCypressTestScript": {
      "description": " When true, Cypress test script will be added to package.json",
      "type": "boolean",
      "$default": {
        "$source": "argv",
        "index": 1
      },
      "x-prompt": "Would you like to add Cypress test script to package.json?"
    },
```

Running cypress test command by angular cli, we already have changes in place in angular.json. hence ```ng e2e``` or ```ng run {project name}:cypress-run```- both works as well.

File: src/schematics/cypress/schema.json [hunk 1]
```diff
@@ -13,6 +13,15 @@
       },
       "x-prompt": "Would you like to remove Protractor from the project?"
     },
+    "addCypressTestScript": {
+      "description": " When true, Cypress test script will be added to package.json",
+      "type": "boolean",
+      "$default": {
```
Comment (anthonymjones) [inline src/schematics/cypress/schema.json]: Please remove the `$default` and `x-prompt` properties here. This option should only be true if the user adds `--addCypressTestScripts` when adding the schematic.

File: src/schematics/cypress/schema.json [hunk 2]
```diff
@@ -13,6 +13,15 @@
       },
       "x-prompt": "Would you like to remove Protractor from the project?"
     },
+    "addCypressTestScript": {
```
Comment (anthonymjones) [inline src/schematics/cypress/schema.json]: Please make this plural, since there are two scripts: `addCypressTestScripts`

**Comment by anthonymjones** [comment]
To answer your question on the prompt, you should just be able to use `"default": false` instead of the `"default$": { ... }`

**Comment by NileshPatel17** [comment]
Changes done as requested.

**Comment by anthonymjones** [comment]
Once the other open PR is resolved and merged, I'll cut a new release.

**Pull request merged**